pkgs/development/python-modules/triton/default.nix

```diff
@@ -159,20 +113,16 @@ buildPythonPackage rec {
       export CC=${cudaPackages.backendStdenv.cc}/bin/cc;
       export CXX=${cudaPackages.backendStdenv.cc}/bin/c++;
 
-      # Work around download_and_copy_ptxas()
-      mkdir -p $PWD/triton/third_party/cuda/bin
-      ln -s ${ptxas} $PWD/triton/third_party/cuda/bin
+      export TRITON_PTXAS_PATH=${ptxas}
+      export TRITON_CUOBJDUMP_PATH=${cudaPackages.cuda_cuobjdump}
+      export TRITON_NVDISASM_PATH=${cudaPackages.cuda_nvdisasm}
+      export TRITON_CUDACRT_PATH=${cudaPackages.cuda_nvcc}
+      export TRITON_CUDART_PATH=${cudaPackages.cuda_cudart}
+      export TRITON_CUPTI_PATH=${cudaPackages.cuda_cupti}
```

Comment:
These could be `env = ... // optionalAttrs cudaSupport { TRITON_XXXX = YYY; }`. Also I imagine they'd want `getBin` for `cuobjdump`, `nvdisasm`, and `nvcc`, and `getLib` or `getDev` for the libraries (not sure which)

Reply:
It did compile, and it worked. Hmm but it could be the case the wanned getLib or getDev. I'm unsure.

Reply:
> It did compile, and it worked.

Hmmm I then suspect that setting these to empty strings would've worked as well (except for ptxas) xDD

Because these paths rn resolve into the `.out` outputs, which are almost empty. It must be that cmake picks up the right paths by the usual cmake means, disregarding the python bullshit